template/ut_template/Makefile

```diff
@@ -23,6 +23,7 @@ TOP_DIR := $(ROOT_DIR)
 
 SRC_DIRS = $(ROOT_DIR)/src
 INC_DIRS := $(ROOT_DIR)/../include
+XCFLAGS := -DNDEBUG
```

Comment:
should be off in the template.

Reply:
you mean it should be commented in the template??